docs/getting-started.md

```diff
@@ -1,276 +1,136 @@
+
+### How to: Make a request to the SnapTrade API
+
+When you have an `API Key` and want to start using the SnapTrade API, you will need to use your `API Key` to make requests to the SnapTrade API.
 
-## Register a test user
+1. With your Free `API Key`, enter your `clientId` and `consumerKey` into the corresponding fields of <https://docs.snaptrade.com/reference/API%20Status/ApiStatus_check>.
+2. Press 'Send'.
 
-To create a secure brokerage authorization, you first need to register a user.
+In the box below the 'request' you should receive a response from the SnapTrade API. The <https://docs.snaptrade.com/reference/API%20Status/ApiStatus_check> endpoint is the most basic request you can make to the SnapTrade API.
 
-Call the :api[Authentication_registerSnapTradeUser] endpoint with a `userId` which can be any string as long as it's unique to a user. Upon receiving a 200 response, the user is registered successfully and you should receive a response with a user id and a randomly generated user secret. 
+You can also make this request with any of the SnapTrade SDKs.
 
-You can think of the user secret as a password that you will store on your users’ behalf. This, combined with your consumerKey provides additional security for your users’ data. The user id and user secret need to be passed along to all SnapTrade API endpoints that involve access to user data.
+### SnapTrade SDKs
 
-Example:
+| **Language** | **SDK Link** |
+| --- | --- |
+| Python | [**https://pypi.org/project/snaptrade-python-sdk**](https://pypi.org/project/snaptrade-python-sdk) |
+| TypeScript | [**https://www.npmjs.com/package/snaptrade-typescript-sdk**](https://www.npmjs.com/package/snaptrade-typescript-sdk) |
+| Java | [**https://central.sonatype.com/artifact/com.konfigthis/snaptrade-java-sdk**](https://central.sonatype.com/artifact/com.konfigthis/snaptrade-java-sdk) |
+| Ruby | [**https://rubygems.org/gems/snaptrade**](https://rubygems.org/gems/snaptrade) |
+| C# | [**https://nuget.org/packages/SnapTrade.Net**](https://nuget.org/packages/SnapTrade.Net) |
+| PHP | [**https://packagist.org/packages/konfig/snaptrade-php-sdk**](https://packagist.org/packages/konfig/snaptrade-php-sdk) |
+| Go | [**https://pkg.go.dev/github.com/passiv/snaptrade-sdks/sdks/go**](https://pkg.go.dev/github.com/passiv/snaptrade-sdks/sdks/go) |
 
-```json
-{ "userId": "USERID123", "userSecret": "AYSEY8726837292873" }
-```
+### API Keys FAQ
+
+- **I have lost the consumerKey to my free API Key. What should I do?**
+  - You should reach out to SnapTrade support. If it is compromised, your users are still safe due to also requiring a user secret to get account data or place trades.
+- **I have lost the consumerKey to my paid API Key. What should I do?**
+  - You should reach out to SnapTrade support. If it is compromised, your users are still safe due to also requiring a user secret to get account data or place trades.
```

Comment:
Duplicating this feels like it's wasting reader's time. I'd combine them into one

Reply:
Agree, addressed this